src/evm/contracts/Venus.ts

```
@@ -0,0 +1,219 @@
+
+  prover: TypedContractMethod<[], [string], "view">;
+
+  receiveEvent: TypedContractMethod<
+    [
+      receiptIndex: BytesLike,
+      receiptRLPEncodedBytes: BytesLike,
+      logIndex: BigNumberish,
+      logBytes: BytesLike,
+      proof: BytesLike
+    ],
+    [void],
```

Biome (1.9.4)

[error] void is confusing outside a return type or a type parameter.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)